packages/development-proxy/preset.js

```diff
@@ -12,10 +12,18 @@ module.exports = {
           patternProperties: {
             '^/': {
               oneOf: [
+                { type: 'string', pattern: '^$' },
```

Comment:
Have you considered using `maxLength: 0` as opposed to the RegExp? Validation error output might be a bit more self-explanatory.